APMrover2/control_modes.cpp

```diff
@@ -1,5 +1,7 @@
 #include "Rover.h"
 
+static const int16_t CH_7_PWM_TRIGGER = 1800;
+
 void Rover::read_control_switch()
```

Comment:
@gmorph static const got plenty advantage again define, I will let you google the difference, you will fine better answer than what I could write. I am surprised that it require a little more ram